libraries/stdlib/jvm/src/kotlin/io/files/FileTreeWalk.kt

```diff
@@ -21,29 +35,29 @@ public enum class FileWalkDirection {
     /** Depth-first search, directory is visited BEFORE its files */
     TOP_DOWN,
     /** Depth-first search, directory is visited AFTER its files */
-    BOTTOM_UP
-    // Do we want also breadth-first search?
+    BOTTOM_UP;
 }
 
 /**
  * This class is intended to implement different file traversal methods.
  * It allows to iterate through all files inside a given directory.
  *
- * Use [File.walk], [File.walkTopDown] or [File.walkBottomUp] extension functions to instantiate a `FileTreeWalk` instance.
+ * Use [File.walk], [File.walkTopDown],[File.walkBottomUp],[File.walkBottomUpWithAlgorithm] or [File.walkTopDownWithAlgorithm] extension functions to instantiate a `FileTreeWalk` instance.
```

Comment:
Nit: Use a space after `,` and hard-wrap line at 120 chars.

Reply:
Agree.

Reply:
> Nit: Use a space after `,` and hard-wrap line at 120 chars.

Done.